File: coding-practice/50 Coding Challenges part 1/Python/problem11.py
# calculate the average numbers of the numbers in an array of numbers
def calculate_average(array):
    sum = 0
    for element in array:
        sum += element
    average = sum / len(array)
    return average


array = [1, 3, 9, 15, 90]
print('The average numbers of that particular array is {}'.format(calculate_average(array)))

File: coding-practice/50 Coding Challenges part 1/C#/problem11.cs
// coding challenges 11 : calculate the average of the numbers in an array of numbers
using System;
class problem11
{
    static float average_array(int[] array){
        float sum = 0f;
        for (int i = 0; i < array.Length; i++)
        {
            sum += array[i];
        }
        return sum/array.Length;
    }
    static void Main(string[] args){
        int[] array = {1, 3, 9, 15, 90};
        float average = average_array(array);

        Console.Write("The average of this array is " + average);
    }
}